.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["dbaeumer.vscode-eslint"]
+}
```

Comment:
please remove auto-generated code from your vscode

Reply:
Neither the flake file nor the ide extension metadata are autogenerated, they're very helpful to get productive without running into a bunch of linting errors etc when the editors are not aware of what sort of codebase is being dealt with, as well as for being able to switch between eslint/non-eslint repos. The flake stuff is for running a shell with node/yarn available, very useful for running an isolated dev environment inside the repo.

Just for some context on why I used them here, nothing overrides the user's editor setup an are just hints if they have no formatting etc to align with your eslint rules! 

Will remove them for now and can merge them back again if you see the value :)